From 6061f7928387c5eec69ae99f8b23cd37ad020288 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Tue, 22 Oct 2019 17:12:24 +0800
Subject: [PATCH] arm64: dts: rockchip: Add boot mode panic for rk3399

Add BOOT_PANIC flag for system reboot from panic.
We can get the boot mode from sys/kernel/boot_mode.

Change-Id: I75ce678eff7b9dc8db80e2d88ea79d888a0a2e2c
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index bc8980f0c5a0..2f8201b4f57b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -1164,6 +1164,7 @@
 			mode-fastboot = <BOOT_FASTBOOT>;
 			mode-loader = <BOOT_BL_DOWNLOAD>;
 			mode-normal = <BOOT_NORMAL>;
+			mode-panic = <BOOT_PANIC>;
 			mode-recovery = <BOOT_RECOVERY>;
 			mode-ums = <BOOT_UMS>;
 		};
-- 
2.35.3

